### ISSUE TITLE
[HelpHub] First Steps with WordPress (Block Editor) - screenshots and alt text

### ISSUE THREAD

**Issue by estelaris** [WordPress/Documentation-Issue-Tracker#1136]
## Issue Description

Needs updated screenshots and alt text is missing

## URL of the Page with the Issue

https://wordpress.org/documentation/article/first-steps-with-wordpress-block-editor/

## Section of Page with the issue

will mention below with screenshots

## Why is this a problem?

outdated images

## Suggested Fix

see below

**Comment by toyinogunseinde** [comment]
I'll work on this

**Comment by toyinogunseinde** [comment]
### Log In Section
No need to change the screenshot
Alt text: Login screen for default WordPress installation

### Start at the top section
Image:
<img width="754" alt="image" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/105225741/6864ecbb-b5f7-4067-a81c-9baa0db60292">

Alt Text: Welcome screen for WordPress 6.3

### Test drive your WordPress site
Alt Text: Default homepage of the Twenty Twenty Two WordPress theme.

### Test Drive the WordPress Administration Screens Section

Image: <img width="754" alt="image" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/105225741/6864ecbb-b5f7-4067-a81c-9baa0db60292">

Alt Text: Welcome screen for WordPress 6.3

### Using WordPress Plugins

Image: 
<img width="745" alt="image" src="https://github.com/WordPress/Documentation-Issue-Tracker/assets/105225741/6de82ebe-e08f-4b2a-bb3a-08f2d791d778">

Alt Text: Plugins page for WordPress
